tests/datasets/test_in_context_learning_datasets.py

```diff
@@ -24,7 +24,8 @@
 from composer.models import HuggingFaceModel
 from composer.trainer import Trainer
 from composer.utils import dist, reproducibility
-from tests.common import device, world_size
+
+# from tests.common import device, world_size
```

Comment:
I commented this out bcus they never ran. They just made all the tests get skipped